contracts/StakeManager.sol

```diff
@@ -39,7 +40,7 @@ contract StakeManager is IStakeManager {
         StakeInfo storage info = stakes[relayManager];
         require(info.owner == msg.sender, "not owner");
         require(info.withdrawBlock == 0, "already pending");
-        info.withdrawBlock = block.number + info.unstakeDelay;
+        info.withdrawBlock = block.number.add(info.unstakeDelay);
```

Comment:
unstakeDelay is time (expressed in 15-second block time). it shouldn't be uint256, but uint128

Reply:
Playing with int size is not security IMHO

Reply:
a better option is `require( unstakeDelay<1e10)` - when adding it. if the number is known to be in valid range, no need to check it over again (both here and in "unauthorize")